autoscheduler/frontend/src/package.json

```diff
@@ -6,18 +6,25 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "dev": "webpack --mode development ./index.tsx --output ../static/main.js",
-    "build": "webpack --mode production ./index.tsx --output ../static/main.js"
+    "build": "webpack --mode production ./index.tsx --output ../static/main.js",
+    "lint": "eslint .",
+    "lint-init": "eslint --init"
```

Comment:
Do we need to include `lint-init`, if we already are including the `.eslintrc` file?

Reply:
No, sorry, I only needed that to generate the config file